foundation/scheduler/scheduler.go

```diff
@@ -54,8 +54,8 @@ func (r *Scheduler) Schedule(s Schedule) {
 	r.core.watch(s)
 }
 
-// Register registers a Runnable with the Scheduler and returns a shortcut function to run those jobs
-func (r *Scheduler) Register(jobType string, runner Runnable, options ...Option) JobFunc {
+// Register registers a Module with the Scheduler and returns a shortcut function to run those jobs
```

Comment:
Same here. Basically all mentions of `Runnable` within the `scheduler` package can be reverted.